docs/source/share_kedro_viz.md

```diff
@@ -23,10 +23,13 @@ When prompted for a project name, you can enter anything, but we will assume `Sp
 
 When your project is ready, navigate to the root directory of the project.
 
-## Update and install the dependencies
+## Publish and share Kedro-Viz automatically
 
-Kedro-Viz requires specific minimum versions of `fsspec[s3]`, and `kedro` to publish your project.
+You can automate the process of publishing and sharing your Kedro-Viz. Currently we support AWS, but other cloud providers, namely Azure and GCP, are in pipeline. [There is a manual, platform-agnostic publish and share process described below](#platform-agnostic-sharing-with-kedro-viz) for static website hosts like GitHub pages, and cloud providers like Azure and GCP.
 
+This section describes the steps to follow to automate your Kedro-Viz deployment to AWS.
+
+### Update and install the dependencies
```

Comment:
It would -- I don't think that was my commit that removed it (apologies if so). If you removed it @rashidakanchwala could you confirm if we should put it back?